examples/tensorizer/serialize_and_load.py

```
@@ -0,0 +1,77 @@
+
+def upload_to_s3(path: str, bucket: str, key: str) -> None:
+    import boto3
+
+    s3 = boto3.client("s3")
+    s3.upload_file(path, bucket, key)
+
+
+def serve_file(path: str, port: int) -> threading.Thread:
+    directory = os.path.dirname(os.path.abspath(path))
+    os.chdir(directory)
+    server = ThreadingHTTPServer(("0.0.0.0", port), SimpleHTTPRequestHandler)
```

Ruff (0.12.2)

Possible binding to all interfaces

(S104)